lib/galaxy/model/mapping.py

```diff
@@ -2783,7 +2783,7 @@ def rating_mapping(rating_class, **kwds):
     output_datasets=relation(model.JobToOutputDatasetAssociation, lazy=True),
     output_dataset_collection_instances=relation(model.JobToOutputDatasetCollectionAssociation, lazy=True),
     output_dataset_collections=relation(model.JobToImplicitOutputDatasetCollectionAssociation, lazy=True),
-    post_job_actions=relation(model.PostJobActionAssociation, lazy=False),
+    post_job_actions=relation(model.PostJobActionAssociation),
```

Comment:
I assume you didn't mean for this to be in here?

Reply:
Indeed, thanks for the catch!